client/rpc-api/src/policy.rs

```diff
@@ -56,7 +56,7 @@ impl std::fmt::Display for UnsafeRpcError {
 impl std::error::Error for UnsafeRpcError {}
 
 impl From<UnsafeRpcError> for rpc::Error {
-	fn from(_: UnsafeRpcError) -> rpc::Error {
-		rpc::Error::method_not_found()
+	fn from(error: UnsafeRpcError) -> rpc::Error {
+		rpc::Error { code: rpc::ErrorCode::MethodNotFound, message: error.to_string(), data: None }
```

Comment:
We should really emit another error code for this.

The method is still found but denied because it's unsafe. 
Perhaps out of scope for this PR...